gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -241,10 +241,11 @@ def run(self) -> GtsfmData:
             client = Client(cluster)
 
         # create process graph
-        process_graph_generator = ProcessGraphGenerator()
-        if isinstance(self.scene_optimizer.correspondence_generator, ImageCorrespondenceGenerator):
-            process_graph_generator.is_image_correspondence = True
-        process_graph_generator.save_graph()
+        # TODO(Handle this)
+        # process_graph_generator = ProcessGraphGenerator()
+        # if isinstance(self.scene_optimizer.correspondence_generator, ImageCorrespondenceGenerator):
+        #     process_graph_generator.is_image_correspondence = True
+        # process_graph_generator.save_graph()
```

Comment:
@stepanyanhayk How should I handle having multiple correspondence generators?

Reply:
Why is this commented out?

Reply:
When we have multiple correspondence generator, I cannot use the single flag to control the process graph generation. I need to upgrade the process graph generator too.